content/07.pathogenesis.md

```diff
@@ -90,8 +90,13 @@ This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were les
 However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
 These differences are present when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever as a presenting symptom for patients upon hospital admission.
 For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients on presentation, while another study [@doi:10.1056/NEJMc2010419] reported cough, fever, and dyspnea as the most common presenting symptoms.
+
+Patients may also experience loss of smell, myalgias (muscle aches), fatigue, or headache.
```

Comment:
We looked a fair bit at this in https://www.medrxiv.org/content/10.1101/2020.06.09.20126813v1 - the paper is accepted and should come out reasonably soon. Maybe we should use that as a reference? Anosmia was a good predictor of a positive result among the folks reporting a test.

Reply:
It's an important distinction that something that is a good predictor of COVID-19 but that is not specific to it would not be a strong predictor in Fig 3 because it's based on the outcome of a test being positive. Based on the time of those survey responses and the limits in US test infrastructure, people probably often needed to have at least some symptoms to receive a test in the first place.

Reply:
It may be good to add this nuance below the sentence that I just added - see what you think @byrdjb!